src/main/java/com/klarna/hiverunner/io/IgnoreClosePrintStream.java

```diff
@@ -18,11 +18,12 @@
 
 import java.io.OutputStream;
 import java.io.PrintStream;
+import java.nio.charset.StandardCharsets;
 
 public class IgnoreClosePrintStream extends PrintStream {
 
     public IgnoreClosePrintStream(OutputStream out) {
-        super(out);
+        super(out, false, StandardCharsets.UTF_8);
```

Comment:
Can you add a comment here explaining what the `false` is for?

Reply:
reverted

Reply:
So for some reason you needed to disable the `autoFlush`? Or is that the default and there was no other super constructor which allowed you to pass in just the `encoding`?

Reply:
Default was false. Added only UTF-8 as it was throwing NPE.